network-api/networkapi/buyersguide/views.py

```diff
@@ -14,6 +15,8 @@
 
 vote_throttle_class = UserVoteRateThrottle if not settings.TESTING else TestUserVoteRateThrottle
 
+path_regex = re.compile(r"^/\w\w/")
```

Comment:
do we just want `\w\w` or do we want `[a-z]{2}(-[A-Z]{2})?`

Reply:
This project only supports [these locales](https://github.com/mozilla/foundation.mozilla.org/blob/8be51a581f3696173dd729e9634a1cec1099e5f6/network-api/networkapi/settings.py#L333-L340) at this time, so we don't need to worry about anthing more complicated than a two-character code. I can make them `[a-z]{2}` but it would accomplish the same thing

Reply:
Go for the more verbose one now so it doesn't trip us up in the future, for the same reason Pomax describes.